### PR TITLE
Feature/main page

### PR DESCRIPTION
**Pull request by VanderbiGit**
Add:
Header 
Router for page
Main page

### CODE REVIEW

File: client/src/components/NavigationBar.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Nav, Navbar } from 'react-bootstrap';
+
+
+export const NavigationBar = () => (
```
Comment (vitaliimak) [inline client/src/components/NavigationBar.js]: Could you replace all arrow functions in components and pages into ES6 classes, as you did it in App.js https://github.com/airbnb/javascript/tree/master/react#class-vs-reactcreateclass-vs-stateless

File: client/src/style/App.css [hunk 2]
```diff
@@ -0,0 +1,10 @@
+body{
+    background-color: white 
+
```
Comment (vitaliimak) [inline client/src/style/App.css]: Is this blank lines need?

**Review by vitaliimak — COMMENTED** [review]
When I click in links in navigation bar application completely reload (you can see it developer tools in network tab). Application must be SPA and load only component or page.

File: client/src/style/App.css (hunk 2 above)
Comment (VanderbiGit) [reply client/src/style/App.css]: yes, its start to modify body,  in class we must speak about design main page.

Comment (vitaliimak) [reply client/src/style/App.css]: I mean line 3 and 4. Is it need?

Comment (VanderbiGit) [reply client/src/style/App.css]: empty line?

Comment (vitaliimak) [reply client/src/style/App.css]: yes

Comment (VanderbiGit) [reply client/src/style/App.css]: no

File: client/src/components/NavigationBar.js (hunk 1 above)
Comment (VanderbiGit) [reply client/src/components/NavigationBar.js]: +

File: client/src/style/App.css (hunk 2 above)
Comment (VanderbiGit) [reply client/src/style/App.css]: +

**Review by vitaliimak — APPROVED** [review]
Well done!

**Review by r-yevchuk — APPROVED** [review]

**Review by Melnyk9506 — COMMENTED** [review]
ok

**Review by Melnyk9506 — APPROVED** [review]
ok

**Pull request merged**